### PR TITLE
feat(navBar): add disabled attribute

### LINKED ISSUE #9667: `md-nav-item` ignores `disabled`-attribute
**Actual Behavior**:
-  `What is the issue? *` `md-nav-item` ignores `disabled`-attribute
-  `What is the expected behavior?` shown in the demo as manual disabled

**CodePen** (or steps to reproduce the issue): *
-  `CodePen Demo which shows your issue:` http://codepen.io/anon/pen/ozBBya
-  `Details:` notice the difference between the two tabs "Attempt to disable" and "Manual disabled"

**Angular Versions**: *
-  `Angular Version:` 1.5.5
-  `Angular Material Version:` 1.1.0

**Additional Information**:
-  `Browser Type: *` tested on desktop, but assume all
-  `Browser Version: *` tested firefox and chrome, but assume all
-  `OS: *`tested on Linux, but assume all
-  `Stack Traces:` no

### PR DESCRIPTION
**Pull request by maxjoehnk**
* Add disabled attribute to navitem
* Update demo to showcase ng-disabled and disabled usage
* Add tests
* Ignore navbar specific button styling when disabled

fixes #9667

### CODE REVIEW

File: src/components/navBar/demoBasicUsage/index.html [hunk 1]
```diff
@@ -28,5 +32,8 @@
 
     <md-checkbox ng-model="disableInkBar">Disable Ink Bar</md-checkbox>
 
+    <md-checkbox ng-model="secondTabDisabled" aria-label="Disable item two?" style="margin: 5px;">
+      Disable Second Tab?
```
Comment (DevVersion) [inline src/components/navBar/demoBasicUsage/index.html]: I think it would look better to remove the `?`  here (also in the `aria-label`)

File: src/components/navBar/navBar.js [hunk 2]
```diff
@@ -442,6 +444,7 @@ function MdNavItem($$rAF) {
       'mdNavSref': '@?',
       'srefOpts': '=?',
       'name': '@',
+      'disabled': '@?'
```
Comment (DevVersion) [inline src/components/navBar/navBar.js]: Why is this necessary, if we already use the `attrs.$observe`?

Comment (maxjoehnk) [inline src/components/navBar/navBar.js]: Yeah, you're right. It isn't necessary.

**Comment by kara** [comment]
@maxjoehnk  Can you rebase?

**Comment by maxjoehnk** [comment]
@kara Done

**Comment by ThomasBurleson** [comment]
@maxjoehnk - something bad happened with your rebase. We now have 40 files to merge. That will not be allowed. Please resubmit a new PR based on the lastest from https://github.com/angular/material `master` branch.
